e2e/ctl_v3_lease_test.go

```diff
@@ -73,6 +75,43 @@ func leaseTestGrantLeasesList(cx ctlCtx) {
 	}
 }
 
+func leaseTestTimeToLiveExpired(cx ctlCtx) {
+	err := leaseTestTimeToLiveExpire(cx, 3)
+	if err != nil {
+		cx.t.Fatal(err)
+	}
+}
+
+func leaseTestTimeToLiveExpire(cx ctlCtx, ttl int) error {
+	leaseID, err := ctlV3LeaseGrant(cx, ttl)
+	if err != nil {
+		return err
+	}
+
+	if err = ctlV3Put(cx, "key", "val", leaseID); err != nil {
+		return fmt.Errorf("leaseTestTimeToLiveExpire: ctlV3Put error (%v)", err)
+	}
+	// eliminate false positive
+	time.Sleep(time.Duration(ttl+1) * time.Second)
+	cmdArgs := append(cx.PrefixArgs(), "lease", "timetolive", leaseID)
+	proc, err := spawnCmd(cmdArgs)
+	if err != nil {
+		return err
+	}
+	line, err := proc.Expect(" granted with TTL(")
```

Comment:
@hexfusion @gyuho 

eventually we should return things like `lease already expired` to explicitly return the error.